integer/integer_test.go

```diff
@@ -242,3 +245,27 @@ func TestRoundToInt32(t *testing.T) {
 		}
 	}
 }
+
+func TestMin(t *testing.T) {
+	if min := Min(math.MinInt, 0); min != math.MinInt {
```

Comment:
Should we have Uint test cases?  I know it's kind of silly...

Reply:
Eh, doesn't hurt. Added.